### DISCUSSION TITLE
Hard Links don't copy the "Date Added" sort from stable

### DISCUSSION THREAD

**Discussion by SupDos** [ppy/osu#21864] [Ideas] [answered]
Importing songs with stable's Date Added has already been asked already, #13713, but I was hoping that Hard Links would do this by default somehow, especially now that people coming from stable to try lazer out with these hard links would expect their beatmap list to have the same order that stable's does 

From what I can see, it doesn't either, it actually seems to add them in the same order as a file copy import would.

Is this something that when added to normal copy import, would work on Hard Link import as well?

**Comment by frenzibyte** [comment, marked as answer]
Yes. The type of the file should not affect copying the "date added" values from stable. Even if, putting this as a heads-up comment in the thread would be 10 times better than opening a separate discussion thread that's currently not linked to the issue thread in any shape or form.
